### PR TITLE
Matt/master/random element

### PR DESCRIPTION
**Pull request by MareoRaft**
PR for #27 

`randomElement` method created.  Usage: `A.randomElement()`.

I was not able to test that the element chosen is truly chosen "at random", nor could i test that the randomness is uniform.  I think some special testing tools would be necessary for this.  That being said, I don't think it's something to worry about.  I'm confident that the function does indeed work correctly.  You can verify yourself that the formula used here matches the formula given in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#Getting_a_random_integer_between_two_values_inclusive, which guarantees a random choice with a uniform distribution.

### CODE REVIEW

File: package/lib/index.js [hunk 1]
```diff
@@ -313,9 +313,22 @@ function any () {
...
 }
-
 global.Set.prototype.any = any
 
+/**
+ * Returns a random element of this set.
+ * One element of this set is chosen at random and returned.  The probability distribution is uniform.  Math.random() is used internally for this purpose.
+ * @function
+ * @name Set.prototype.randomElement
+ * @returns {*} An element chosen randomly from the current set that could by of any type, depending on the elements of the set.
```
Comment (jankapunkt) [inline package/lib/index.js]: Typo in " could by of any type"

**Review by jankapunkt — CHANGES_REQUESTED** [review]
Implentation is fine. Can be merged once the typo is fixed.

File: package/lib/index.js (hunk 1 above)
Comment (MareoRaft) [reply package/lib/index.js]: typo now fixed

**Comment by MareoRaft** [comment]
merge conflicts fixed.  ok to merge once Travis succeeds.

**Review by jankapunkt — APPROVED** [review]

**Pull request merged**